src/emailpro/account/emailpro-account.html

```diff
@@ -226,11 +226,18 @@
         <div class="col-md-3 mt-5 mt-lg-0"
              data-ng-if="exchange.isMXPlan">
             <button class="btn btn-block btn-default" type="button"
+                        data-ng-if="accountsConfigured.length < exchange.accountsNumber"
                         data-ng-click="addNewConfigureAccount()"
                         data-ng-disabled="loadingNewConfiguredAccount">
                 <oui-spinner size="s" data-ng-show="loadingNewConfiguredAccount"></oui-spinner>
                 <span data-translate="emailpro_ACTION_add_account_button"></span>
             </button>
+            <a
+                data-ng-if="accountsConfigured.length === exchange.accountsNumber && !noDomainFlag"
+                data-ng-href="{{ accountOrder }}"
```

Comment:
```suggestion
                data-ng-href="{{:: accountOrder }}"
```
looks like it never changes once it's done